src/globals/globals.ts

```diff
@@ -0,0 +1 @@
+import '@esri/calcite-components/dist/index.js'
```

Comment:
This seems to be the critical thing. This file seems to simply need to exist.

Reply:
Further testing reveals that this file can be called anything and doesn't even need to be in the globals directory (just in src, I assume). So I guess the compiler is bundling all the src files....